src/Parser/AfflictionWarlock/Modules/WarlockCore/getDamageBonus.js

```diff
@@ -0,0 +1,3 @@
+export default function getDamageBonus(event, increase) {
```

Comment:
This doesn't include the absorbed damage, is this on purpose?

I'm a believer in including absorbed damage as if it were regular damage; the bosses where this has been relevant lately this has always been useful towards killing the boss, so damage contributions towards that help progression so it's pretty useful.